src/main/java/uk/gov/hmcts/reform/bsp/common/mapper/AddressMapper.java

```diff
@@ -12,6 +12,25 @@
 
 public class AddressMapper {
 
+    public static class Fields {
```

Comment:
I think this class doesn't need to be static. Happy to keep it static if you prefer that, though.